README.md

```diff
@@ -1,5 +1,7 @@
 # A sample Python project
 
+![alt text](https://www.python.org/static/img/python-logo.png "Sample inline image")
```

Comment:
```suggestion
![Python Logo](https://www.python.org/static/community_logos/python-logo.png "Sample inline image")
```
The original is designed for a dark background, but this will almost always be on a white background. Also adds the alt text.